src/main.cpp

```diff
@@ -61,7 +61,23 @@ int main(int argc, char * argv[]) {
         args.printUsage();
         return 0;
     }
-
+    if(args.getSettingsPathSet()) {
+        QDir settingsPath(args.getSettingsPath());
+        qDebug() << "Settings Path Provided: " << settingsPath.absolutePath();
+        if(!args.getSettingsPathStatus()) {
+            qDebug() << "The provided settings path does not have the proper permissions";
+            QApplication app(argc, argv);
+            QMessageBox::critical(0, QObject::tr("Cannot access this path"),
+                                     QObject::tr("The Settings Path: %1 that is "
+                                        "provided does not have the "
+                                        "proper permission to access its "
```

Comment:
so for this case, I need to remove the test at https://github.com/mixxxdj/mixxx/pull/1086/files#diff-7ec3c68a81efff79b6ca22ac1f1eabbaR64 , as this checks if the new path, not taking the default ~/.mixxx path in the consideration